### ISSUE TITLE
Make it clear what OAuth 2.0 flows are supported

### ISSUE THREAD

**Issue by IGassmann** [nuxt-community/auth-module#691]
It's currently not clear in the documentation what OAuth 2.0 flows are supported. For instance, the Authorization Code flow is only supported with the PKCE extension.

**Comment by kevintechie** [comment]
I agree that the docs should be explicit and show how to set up each flow.

However, I'm currently using Authorization Code flow without PKCE with nuxt-auth. Unless something has changed recently, it should work.
